src/glossary.adoc

```diff
@@ -111,6 +111,12 @@ A macro reference that consists of a module name and either a macro name exporte
 or a numeric address within the range of the module’s exported macro table. In text, these look
 like `:__module-name__:__name-or-address__`.
 
+quasi-literal::
+A template, denoted as a list or struct, that is _partly_ (“quasi-”) literal.
+List-shaped templates treat the elements as nested templates.
+Struct-shaped templates treat the field names as literal, but the corresponding values as
+templates.
```

Comment:
I suggest adding a sentence mentioning that S-expressions are _not_ quasi-literals; listing two out of the three container types naturally raises the question.

Reply:
Added.